### DISCUSSION TITLE
Add more sponsorship options?

### DISCUSSION THREAD

**Discussion by leo150** [mikebrady/shairport-sync#1601] [Ideas]
Looks like PayPal doesn't support recurring payments. Patreon or native GitHub option would be great.

**Comment by mikebrady** [comment]
Thanks for the suggestions. Just FYI, the native GitHub system is unattractive to recipients outside the U.S., as it requires them to fill U.S. tax returns.

**Reply by leo150** [reply]
Hey. Any update on this? To have Patreon, for instance.

**Comment by mikebrady** [comment]
Thanks for the reminder. Looking at Patreon now and also, again, GitHub.

**Comment by mikebrady** [comment]
Okay, so I've re-enabled the native GitHub sponsorship -- the requirement to fill out U.S. tax returns is gone, thankfully.
